### ISSUE TITLE
DSL debug

### ISSUE THREAD

**Issue by fornellas** [facebookincubator/TestSlide#89]
Sometimes it is hard to understand what hooks (before / after / around) are being executed. It would be nice to have a --debug option that would make the test runner show what hook is being executed, something like this perhaps:

```
Some Context
  before: do_something() file.py:33
    Some Example
      example: some_example() file.py:38
      PASS
  after: cleanup() file.py:45
```
